src/core/server/logging/layouts/json_layout.test.ts

```diff
@@ -94,7 +94,7 @@ test('`format()` correctly formats record with meta-data', () => {
       })
     )
   ).toStrictEqual({
-    ecs: { version: '1.9.0' },
+    ecs: { version: '1.12.0' },
```

Comment:
Maybe we should use `expect.any(String)` assertion to reduce flakiness? A single assertion against `ecs.version` is a specialized test suite is enough.